packages/create-astro/src/actions/typescript.ts

```diff
@@ -61,7 +69,8 @@ export async function typescript(ctx: PickedTypeScriptContext) {
 		if (ts === 'relaxed' || ts === 'default') {
 			ts = 'base';
 		}
-		await spinner({
+		ctx.tasks.push({
+			pending: 'Customize TypeScript',
```

Comment:
I'm not totally sure how we should display these tasks, but maybe just the first word? Again, will keep the label from jumping when the task actually starts.
```suggestion
			pending: 'TypeScript',
```

Reply:
The first word seems like a good solution to me!